docs/addons/install-addons.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Set up Storybook addons
+---
+
+Storybook boasts a wide variety of addons in it's ecosystem. All of them packaged as NPM modules and listed in our [addon gallery](/addons). We're going to guide you through the necessary steps to install and register Storybook addons.
+
+### Install Storybook addons
+
+With the exception of presets, all other Storybook addons follow the same set of installation instructions. For example, to include accessibility testing in Storybook, run the following command to install the necessary addon:
+
+```shell
+yarn add -D @storybook/addon-a11y
+```
+
+### Addon registration
+
+Installing the addon is not enough, it needs to be registered. Update [`.storybook/main.js`](../configure/overview.md#configure-story-rendering) to the following:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-main-addon-registration.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<div class="aside">
+Other addons like the one mentioned might require additional configurations. Check the addon README for additional configuration requirements.
+</div>
+
+That's it, when Storybook starts up accessibility testing is enabled.
+
+![Storybook addon installed and registered](./storybook-addon-installed-registered.png)
+
+
+### Install Storybook presets
```

Comment:
Let's use consistent terminology and call them `Preset addons`

Reply:
Updated with the latest commit